src/values/expressions/SequenceExpression.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Extractor function for a SequenceExpression type value node.
+ * A Sequence expression is an object with an attribute named 
+ * expressions which contains an array of different types 
+ *  of expression objects.
+ *
+ * @returns - An array of the extracted elements.
+ */
+export default function extractValueFromSequenceExpression(value) {
+  // eslint-disable-next-line global-require
+  const getValue = require('./index.js').default;
```

Comment:
```suggestion
  const getValue = require('./').default;
```
(i realize there's other files that include the redundant "index.js" but let's not continue the mistake)

Reply:
Yarn test gave me this error:
"  11:28  error  Useless path segments for "./", should be "."  import/no-useless-path-segments"

So ended up with: const getValue = require('.').default;